src/components/safe-apps/AppCard/index.tsx

```diff
@@ -92,10 +92,19 @@ const AppCardContainer = ({ url, children, variant }: AppCardContainerProps): Re
   )
 }
 
-const CompactAppCard = ({ url, safeApp, onPin, pinned }: CompactSafeAppCardProps): ReactElement => (
+const CompactAppCard = ({ url, safeApp, onPin, pinned, onShareClick }: CompactSafeAppCardProps): ReactElement => (
   <AppCardContainer url={url} variant="compact">
     <div className={styles.compactCardContainer}>
       <img src={safeApp.iconUrl} style={{ width: 52, height: 52 }} alt={`${safeApp.name} logo`} />
+      <IconButton
+        aria-label={`Share ${safeApp.name}`}
+        size="small"
+        onClick={onShareClick}
+        title={`Copy share URL for ${safeApp.name}`}
+        sx={{ width: '32px', position: 'absolute', top: 2, left: 2 }}
+      >
+        <ShareIcon width={16} alt="Share icon" />
+      </IconButton>
```

Comment:
We are missing a UI feedback when the user presses the button. In safe-react we had a notification pop up but I could also imagine handling it the same way we do when copying the safe address from the sidebar by showing a tooltip. Wdyt?

Reply:
Nice, the tooltip works great! I think in the process the share button got oval shaped though:

<img width="81" alt="Screenshot 2022-09-09 at 12 02 49" src="https://user-images.githubusercontent.com/5880855/189325599-c78f4611-967c-4189-ae42-b1fbf8fd391b.png">

Reply:
Fixed now